### DISCUSSION TITLE
how do you create a trip or route between a start location and finish location?

### DISCUSSION THREAD

**Discussion by David-Levinthal** [jaybo/deepzoom-blog#33] [Q&A] [answered]
Having just started using deepzoom, I find it is not quite clear how to creat a "trip". Is it possible to just declare the start and finish and have the course of the trip created? How does one create a trip from say gig harbor wa to oak harbor wa? Or is what I am asking for a route?

**Comment by jaybo** [comment]
A trip is just a container, file, cloud storage location, url - different ways of saying the same thing.
A trip can contain routes, markers, tracks, settings, and scripts.

DeepZoom doesn't do auto routing.  But it's pretty easy to plop down a series of waypoints to create a route.
Start with this tool:

![image](https://github.com/jaybo/deepzoom-blog/assets/1607323/59f14e06-9b19-4062-bd61-2c2f97eebde2)

**Reply by David-Levinthal** [reply, marked as answer]
I did figure it out..eventually  :-)
